src/test/java/picard/sam/FastqToSamTest.java

```diff
@@ -177,6 +181,67 @@ public void testEmptyFastqAllowed() throws IOException {
         convertFile(emptyFastq, null, FastqQualityFormat.Illumina, false, false, true);
     }
 
+    @Test
+    public void testStreamInputWithoutQuality() throws IOException {
+        ByteArrayOutputStream stderrStream = new ByteArrayOutputStream();
+        PrintStream newStderr = new PrintStream(stderrStream);
+        PrintStream oldStderr = System.err;
+        System.setErr(newStderr);
```

Comment:
Could you wrap this in a try/finally block so the stream reset happens even in the case of an unexpected exception.

Reply:
Done